src/fuzzer.rs

```diff
@@ -49,6 +64,525 @@ pub async fn new(urls: Vec<String>, browser: Browser, opt: parser::Options) {
 
 				eprintln!("{}", msg)
 			}
+
+			if let Some(p) = page {
+				p.close().await.unwrap();
```

Comment:
same thing, this blocks until the page is actually closed and prevents progress on the scans.